examples/example2/script.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Initialize Mollie Components instance
+ */
+const mollie = Mollie("pfl_saaa222aaa3", {
+  locale: "en_US"
```

Comment:
Should we add a comment they should only supply this if they know it? Otherwise it would be better to leave detection up to us.

Reply:
Sounds good. I've added a comment